package.json

```diff
@@ -147,7 +147,6 @@
     "shadow-cljs": "2.11.20",
     "style-loader": "^0.19.0",
     "uglifyjs-webpack-plugin": "^1.0.0",
-    "unused-files-webpack-plugin": "^3.0.0",
```

Comment:
# Remove `UnusedFilesWebpackPlugin`
```
warning " > @cypress/webpack-preprocessor@4.1.5" has incorrect peer dependency "webpack@^4.18.1".
warning " > @testing-library/cypress@5.3.1" has incorrect peer dependency "cypress@^2.1.0 || ^3.0.0 || ^4.0.0".
warning " > babel-loader@7.1.5" has incorrect peer dependency "webpack@2 || 3 || 4".
warning " > extract-text-webpack-plugin@3.0.2" has incorrect peer dependency "webpack@^3.1.0".
warning " > html-webpack-plugin@2.30.1" has incorrect peer dependency "webpack@1 || ^2 || ^2.1.0-beta || ^2.2.0-rc || ^3".
warning " > postcss-cssnext@2.11.0" has unmet peer dependency "caniuse-db@^1.0.30000652".
warning " > uglifyjs-webpack-plugin@1.3.0" has incorrect peer dependency "webpack@^2.0.0 || ^3.0.0 || ^4.0.0".
```

Running `yarn-dev-ee` after this produces the following error:
```
[frontend] [webpack-cli] TypeError: compiler.plugin is not a function
[frontend]     at ExtractTextPlugin.apply (/Users/offbynone/code/work/metabase/node_modules/extract-text-webpack-plugin/dist/index.js:147:16)
[frontend]     at createCompiler (/Users/offbynone/code/work/metabase/node_modules/webpack/lib/webpack.js:74:12)
[frontend]     at create (/Users/offbynone/code/work/metabase/node_modules/webpack/lib/webpack.js:123:16)
[frontend]     at webpack (/Users/offbynone/code/work/metabase/node_modules/webpack/lib/webpack.js:147:32)
[frontend]     at WebpackCLI.f [as webpack] (/Users/offbynone/code/work/metabase/node_modules/webpack/lib/index.js:54:15)
[frontend]     at WebpackCLI.createCompiler (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/lib/webpack-cli.js:1845:29)
[frontend]     at Command.<anonymous> (/Users/offbynone/code/work/metabase/node_modules/@webpack-cli/serve/lib/index.js:81:30)
[frontend]     at async Promise.all (index 1)
[frontend]     at Command.<anonymous> (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/lib/webpack-cli.js:1289:13)
error Command failed with exit code 2.
```
Realted reading:
- https://v4.webpack.js.org/plugins/extract-text-webpack-plugin/
> :warning: Since webpack v4 the extract-text-webpack-plugin should not be used for css. Use [mini-css-extract-plugin](https://v4.webpack.js.org/plugins/mini-css-extract-plugin/) instead.